website/content/en/highlights/2020-07-10-add-reduce-transform.md

```diff
@@ -86,7 +86,8 @@ data_dir = "tmp"
   inputs = ["transform1"]
   type = "file"
   path = "output.log"
-  encoding = "ndjson"
+  framing = "newline_delimited"
+  encoding.codec = "json"
```

Comment:
This won't actually work in the release this highlight is for (0.10.0). I think we should leave old release highlights as they are in case users are on older versions.

Reply:
Alternatively we add a comment to the example config with the new config fields with a note about version support.

Reply:
Done with d7e723dc92dacf98c073ce8605701145fcfc14d5